packages/terra-table/src/TableRow.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import TableRowContent from './TableRowContent';
+
+const propTypes = {
+  /**
+   * The children passed to the component
+   */
+  children: PropTypes.node,
+  /**
+   * Whether or not row is selected
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * The maximum height for the row in a table
+   */
+  height: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge']),
+  /**
+   * Whether or not row is selectable
+   */
+  isSelectable: PropTypes.bool,
+};
+
+const defaultProps = {
+  isSelected: false,
+  isSelectable: undefined,
```

Comment:
removed the default for isSelectable. Cannot add default for height as it will set the maximum height on the row and by default we are not setting restrictions on height